example/src/App.tsx

```diff
@@ -4,12 +4,10 @@ import logo from './logo.svg';
 import {Routes, useLocation} from 'react-awesome-router';
 
 const App: React.FC = () => {
-  const {location, context, setLocation, setContext} = useLocation();
+  const {context, setLocation, setContext} = useLocation();
 
   const login = () => {
     setContext({auth: {logued: true, username: 'notadmin'}});
-    //Optional, force refresh on login
-    setLocation(location);
```

Comment:
No longer necessary to do manual refresh, since we are now fully reactive to changes in context.